sklearn/model_selection/_prediction.py

```diff
@@ -0,0 +1,801 @@
+from collections.abc import MutableMapping
```

Comment:
I think this private sub module would better be named `sklearn/model_selection/_classification_threshold.py` instead of `_prediction.py` since to make it clear that it is unrelated to regression or clustering.